src/scripts/Controller.js

```diff
@@ -518,6 +521,25 @@ export default class SitemapController {
 			},
 		});
 	}
+	readBlob() {
+		let files = document.getElementById('sitemapFiles').files;
+		if (!files.length) {
+			alert('Please select a file!');
+			return;
+		}
+		let file = files[0];
+		$('#file_content').hide();
+		let blob = file.slice(0, file.size);
+		blob.text().then(function(text) {
+			document.getElementById('sitemapJSON').value = text;
```

Comment:
Here the same change to jquery interaction with document.  `.val(value)` for value insertion